.github/workflows/cluster_cleanup.yml

```diff
@@ -30,14 +30,18 @@ on:
         description: "Resource group to clean up"
         required: true
         default: ops-cli-int-test-rg
+      keyvault_prefix:
+        type: string
+        description: "Prefix of keyvault to delete"
+        default: "opskv"
+        required: false
   # Run every night at midnight (Pacific) to cleanup resources
-  schedule:
-    - cron: '0 8 * * *'
+  # schedule:
+  #   - cron: '0 8 * * *'
```

Comment:
should we still run this on a schedule?

Reply:
yea - I will update that right before we merge it in